build.gradle

```diff
@@ -37,5 +39,5 @@ android {
 }
 
 dependencies {
-    compile "com.android.support:support-annotations:24.0.0"
+    implementation "androidx.appcompat:appcompat:$ANDROIDX_VERSION"
```

Comment:
But looks that we don't need appcompat. `androidx.annotation:annotation:$ANDROIDX_VERSION` should be enough